backend/app/conversations/botVars.js

```diff
@@ -0,0 +1,25 @@
+
+const botVars = {
+  orgName: {
+    callparty: 'CallParty',
+    govtrack: 'GovTrack',
+  },
+  orgEmail: {
+    callparty: 'hi@callparty.org',
+    govtrack: 'hi@govtrack.org'
+  }
+}
+
+// this mapping from bot to botType allows testing bots to share botVars with prod bots more easily
+const botTypes = {
+  callparty: 'callparty',
+  callingteststaging: 'callparty',
+  callparty5: 'callparty',
```

Comment:
I tried to put helpful error messages that will let people add their own bot info as necessary if its missing